test/unit/annotation_layer_spec.js

```diff
@@ -172,4 +172,8 @@ describe('Annotation layer', function() {
       expect(borderStyle.verticalCornerRadius).toEqual(0);
     });
   });
+  describe('VideoAnnotation', function() {
+    // unit tests functions here
```

Comment:
This needs to be filled, but I think you were already aware of that. Just mentioning it for the sake of completeness.